components/hmpo-word-count/macro.njk

```diff
@@ -0,0 +1,7 @@
+{% macro hmpoWordCount(ctx, params, base) %}
```

Comment:
Honestly I don't really think there's a need to expose this as hmpoWordCount / hmpoCharacterCount - I don't see why we don't just expose the text-count that you've added and then the consuming app can just pass in type: wordcount/charactercount

Reply:
That's fair enough